packages/frontend/src/contexts/game/reducer/index.js

```diff
@@ -58,6 +60,15 @@ export const gameReducer = (state, action) => {
         turnCount: state.turnCount + 1,
       };
 
+    case types.HANDLE_GAME_OVER:
+      return {
+        ...state,
+        stage: GAME_STAGES.GAME_OVER,
+        winner: state.players.filter(
+          (player) => player.score === getHighScore(state.players)
+        ),
```

Comment:
The `winner` object could probably just be a single player object instead of an array.

Reply:
Instead of setting the game stage to GAME_OVER here, it could probably be set in the FLIP_TILE reducer or in the HANDLE_MATCH reducer.

Reply:
@perennialAutodidact the winner array stores more than one winner object if it's a tie, so result display component checks for the length to be more than one to display the tie message. In future versions where there will be more than two players, it could be useful to display the players that tied as well.